File: src/main/java/cz/wake/manager/commads/servers/Creative_command.java
package cz.wake.manager.commads.servers;

import co.aikar.commands.BaseCommand;
import co.aikar.commands.CommandHelp;
import co.aikar.commands.annotation.CommandAlias;
import co.aikar.commands.annotation.Default;
import co.aikar.commands.annotation.Description;
import co.aikar.commands.annotation.HelpCommand;
import cz.craftmania.craftlibs.utils.ChatInfo;
import cz.wake.manager.Main;
import org.bukkit.command.CommandSender;
import org.bukkit.entity.Player;

@CommandAlias("creative")
@Description("Připojí tě na Creative")
public class Creative_command extends BaseCommand {

    @HelpCommand
    public void helpCommand(CommandSender sender, CommandHelp help) {
        sender.sendMessage("§e§lCreative commands:");
        help.showHelp();
    }

    @Default
    public void connectToCreative(CommandSender Sender) {
        if (Sender instanceof Player) {
            Player player = (Player) Sender;
            try {
                ChatInfo.INFO.send(player, "Teleportuji na server §fCreative");
                Main.getInstance().sendToServer(player, "creative");
            } catch (Exception e) {
                e.printStackTrace();
                player.sendMessage("§cTeleport na server §fCreative §cse nezdařil!");
                ChatInfo.DANGER.send(player, "Teleport se nezdařil: §fCreative");
                Main.getInstance().sendSentryException(e);
            }
        }
    }
}

File: src/main/kotlin/cz/wake/manager/commands/servers/CreativeCommand.kt
package cz.wake.manager.commands.servers

import co.aikar.commands.BaseCommand
import co.aikar.commands.CommandHelp
import co.aikar.commands.annotation.CommandAlias
import co.aikar.commands.annotation.Default
import co.aikar.commands.annotation.Description
import co.aikar.commands.annotation.HelpCommand
import cz.craftmania.craftlibs.utils.ChatInfo
import cz.wake.manager.Main
import org.bukkit.command.CommandSender
import org.bukkit.entity.Player

@CommandAlias("creative")
@Description("Připojí tě na Creative")
class CreativeCommand : BaseCommand() {

    @HelpCommand
    fun helpCommand(sender: CommandSender, help: CommandHelp) {
        sender.sendMessage("§e§lCreative commands:")
        help.showHelp()
    }

    @Default
    fun connectToCreative(sender: CommandSender?) {
        if (sender is Player) {
            try {
                ChatInfo.INFO.send(sender, "Teleportuji na server §fCreative")
                Main.getInstance().sendToServer(sender, "creative")
            } catch (exception: Exception) {
                exception.printStackTrace()
                sender.sendMessage("§cTeleport na server §fCreative §cse nezdařil!")
                ChatInfo.DANGER.send(sender, "Teleport se nezdařil: §fCreative")
                Main.getInstance().sendSentryException(exception)
            }
        }
    }
}